mediacapture-streams/MediaStreamTrack-getSettings.https.html

```diff
@@ -0,0 +1,50 @@
+<!doctype html>
+<html>
+<head>
+<title>MediaStreamTrack GetSettings</title>
+<link rel="author" title="Harald Tveit Alvestrand" href="mailto:hta@google.com"/>
+<link rel="help" href="http://dev.w3.org/2011/webrtc/editor/getusermedia.html#widl-MediaStreamTrack-id">
+</head>
+<body>
+<p class="instructions">When prompted, accept to share your audio and video stream.</p>
+<h1 class="instructions">Description</h1>
+<p class="instructions">This test checks that track.getSettings has expected results.</p>
```

Comment:
"expected results" is pretty vague - maybe give more details on the actual scope of the test?